### PR TITLE
Stat actions - dropdown and submit

### LINKED ISSUE #2123: Add tracking for Submit and Dropdown

### PR DESCRIPTION
**Pull request by na3d**

### CODE REVIEW

File: components/utils/EmitEvent.js [hunk 1]
```diff
@@ -8,4 +8,5 @@ export default function EmitEvent(eventName, category, action, label, value = nu
       value,
     }
   }));
+  console.log(category, eventName);
```
Comment (eatonphil) [inline components/utils/EmitEvent.js]: Console log

File: src/constants.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-undef */
-export const API_ROOT = ENV_API_ROOT || 'https://api.linode.com/v4';
-export const LOGIN_ROOT = ENV_LOGIN_ROOT || 'https://login.linode.com';
+export const API_ROOT = ENV_API_ROOT || 'https://api.alpha.linode.com/v4';
+export const LOGIN_ROOT = ENV_LOGIN_ROOT || 'https://login.alpha.linode.com';
```
Comment (eatonphil) [inline src/constants.js]: These are no longer public so they should not be the default.

**Comment by mparke** [comment]
@eatonphil is this now handled in another pull?

**Comment by eatonphil** [comment]
Not yet, finishing up testing on the FormModalBody PR first then will bring this into a new PR.

**Comment by eatonphil** [comment]
@mparke actually it depends on that PR being merged first. After that it will simply be a matter of supplying a `title` attribute to the `Form` component.

**Comment by eatonphil** [comment]
This has been superceded by #2189

**Pull request closed without merging**